components/music/cards/Artist.tsx

```diff
@@ -71,6 +73,7 @@ const Artist = async ({
             ${isCompact ? "bottom-0" : "top-0"}
```

Comment:
Make this ``<div className={`card-actions ${isCompact ? "justify-end" : " absolute top-0 right-0 p-2"}`}>`` to keep the feed cards consistent with each other. 

Also change line 52 from `<div className={isCompact ? "p-2" : "pt-8 px-8"}>` to `<div className={isCompact ? "p-2" : "pt-8 px-8 mt-5"}>`

Reply:
Made the suggested changes. The play button should now show up next to the share button for artist cards.